src/utils/DataConfig.py

```diff
@@ -27,11 +32,43 @@ class DataConfig:
     activity_idx_to_activity_name_map = None
     subject_idx_to_subject_name_map = None
 
-    timestep_frequency = None # Hz
+    timestep_frequency = None  # Hz
+    variance = None
+    mean = None
+    DATA_CONFIG_METADATA_FILE = "dataConfigMetadata.json"
 
-    # interface (subclass responsibility to define) ------------------------------------------------------------
+    def __init__(self, dataset_path: str):
+        self.dataset_path = dataset_path
 
     def load_dataset(self) -> "list[Recording]":
+        recordings = self._load_dataset()
+
+        print(
```

Comment:
i am a bit confused by the prints here, you state that calculating might take a while before checking if they are available, then you only print that they where found if you need to calculate them and no message is written if they have been set before